lib/api/fields/functions.php

```diff
@@ -96,12 +96,14 @@ function beans_field( array $field ) {
 		return;
 	}
 
+	$group_field_type = 'group' === $field['type'];
```

Comment:
Use this new variable on [line 133](https://github.com/Getbeans/Beans/pull/197/files#diff-52e7616cfa8a08979201d13f82e45f0fL120) too.